sweep/store_test.go

```diff
@@ -14,35 +14,13 @@ import (
 // TestStore asserts that the store persists the presented data to disk and is
 // able to retrieve it again.
 func TestStore(t *testing.T) {
-	t.Run("bolt", func(t *testing.T) {
-
-		// Create new store.
-		cdb, err := channeldb.MakeTestDB(t)
-		if err != nil {
-			t.Fatalf("unable to open channel db: %v", err)
-		}
-
-		testStore(t, func() (SweeperStore, error) {
-			var chain chainhash.Hash
-			return NewSweeperStore(cdb, &chain)
-		})
-	})
-	t.Run("mock", func(t *testing.T) {
-		store := NewMockSweeperStore()
```

Comment:
Seems like we don'T use the Mock anymore, prob. relates to the question above.

Reply:
yeah this is just a test to check the mock sweeper store🤦 The actual usage of the mock store is in the following PRs.